source/slang/slang-diagnostic-defs.h

```diff
@@ -2930,6 +2930,11 @@ DIAGNOSTIC(
     Error,
     dynamicDispatchOnSpecializeOnlyInterface,
     "type '$0' is marked for specialization only, but dynamic dispatch is needed for the call.")
+DIAGNOSTIC(
+    52009,
+    Error,
+    cannotEmitReflectionWithoutTarget,
+    "cannot emit reflection JSON: no compilation target available")
```

Comment:
This is a minor nit, but I believe this grammatically calls for a semicolon rather than a colon.